packages/react-components/src/AccountSidebar/RegistrarJudgement.tsx

```diff
@@ -34,9 +35,9 @@ const JUDGEMENT_ENUM = [
 ];
 
 const OPT_ID = {
-  transform: (optId: Option<PalletIdentityRegistration>): HexString | null =>
+  transform: (optId: Option<ITuple<[PalletIdentityRegistration]>>): HexString | null =>
```

Comment:
I believe the accurate typing here would be: 

`Option<ITuple<[PalletIdentityRegistration, Option<Bytes>]>>`

Ref: https://github.com/polkadot-js/api/blob/master/packages/api-augment/src/polkadot/query.ts#L745